src/app/projects/projlist-map/projlist-map.component.ts

```diff
@@ -3,6 +3,7 @@ import { ApplicationRef, ElementRef, SimpleChanges, Injector, ComponentFactoryRe
 import { Subject } from 'rxjs';
 import 'leaflet';
 import 'leaflet.markercluster';
+import { vectorBasemapLayer } from "esri-leaflet-vector";
```

Comment:
Appreciate the (hopefully) increased type safety. Also liking the smaller code footprint